t/ui/01-list.t

```diff
@@ -388,6 +416,14 @@ subtest 'check test results of job99940' => sub {
     }
 };
 
+subtest 'display job template name in test' => sub {
```

Comment:
actually I think this should say "display**s**", same in the strings below

Reply:
I would actually rephrase as `job template names displayed on "All tests" page`.